test/fixtures/template-auto-bind.html.out

```diff
@@ -14,7 +14,7 @@
     User says: <span>{{greeting}}</span>
   </div>
   <!--
-      The expression {{greeting + " world"}} can't work in a
+      The expression   greeting + " world"   can't work in a
```

Comment:
Yes, rather than pipe along whether the expression is one-way or not to this warning message I just used spaces to separate out the expression from the rest of the warning.